addons/account/models/account_journal.py

```diff
@@ -125,7 +125,13 @@ def _get_default_account_domain(self):
         help="Company related to this journal")
     country_code = fields.Char(related='company_id.account_fiscal_country_id.code', readonly=True)
 
-    refund_sequence = fields.Boolean(string='Dedicated Credit Note Sequence', help="Check this box if you don't want to share the same sequence for invoices and credit notes made from this journal", default=False)
+    refund_sequence = fields.Boolean(
+        string="Dedicated Credit Note Sequence",
+        help="Check this box if you don't want to share the same sequence for invoices and credit notes made from this journal",
+        compute="_compute_refund_sequence",
+        readonly=False,
```

Comment:
better to have readonly and store together.  No need for default False as it is the default default already.